routes/actions.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+
+const actions = require("../data/helpers/actionModel.js");
+const projects = require("../data/helpers/projectModel.js");
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const theActions = await actions.get();
+    res.status(200).json(theActions);
+  } catch {
+    res.status(500).json({ Error: "Unable to retrieve actions" });
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const theActions = await actions.get(req.params.id);
+    if (theActions) {
+      res.status(200).json(theActions);
+    } else {
+      res.status(400).json({ Error: "The requested action does not exist" });
+    }
+  } catch {
+    res.status(500).json({ Error: "There was a problem retrieving the post" });
+  }
+});
+
+router.post("/", async (req, res) => {
+  try {
+    if (req.body.project_id && req.body.description) {
+      const theProject = await projects.get(req.body.project_id);
+      if (theProject) {
+        const theActions = await actions.insert(req.body);
```

Comment:
Dont forget to return a response in an `else` condition in the case that `req.body` doesn't contain an id or description.